service/tasks/driver.py

```diff
@@ -1149,23 +1147,6 @@ def _deploy_instance(driverCls, provider, identity, instance_id,
         _deploy_instance.retry(exc=exc)
 
 
-def _parse_steps_output(msd):
```

Comment:
overall there are problems with this function.  No clue as to what msd should actually be, though I am guessing that it is the message from the exception.  However this function doesn't return anything, though where is it being called from seems to expect a returned string.